pkg/registry/apis/provisioning/repository/local.go

```diff
@@ -277,12 +277,12 @@ func (r *localRepository) History(ctx context.Context, logger *slog.Logger, path
 
 // Webhook implements Repository.
 func (r *localRepository) Webhook(ctx context.Context, logger *slog.Logger, req *http.Request) (*provisioning.WebhookResponse, error) {
-	return nil, &apierrors.StatusError{
-		ErrStatus: metav1.Status{
-			Code:    http.StatusNotImplemented,
-			Message: "webhook not implemented",
-		},
-	}
+	return &provisioning.WebhookResponse{
```

Comment:
I guess this was a test. Should we revert?

Reply:
It is nice to have something not-github to trigger locally :)